frontend/src/routes/SearchPage/components/HeaderWithNotification.test.tsx

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2021 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+import React from 'react'
+import { render } from '@testing-library/react'
+import HeaderWithNotification from './HeaderWithNotification'
+import { MockedProvider } from '@apollo/client/testing'
+
+///case where we have a message about disabled search (current message)
+test('renders clusters disabled message', () => {
+    const disableSearch = [
+        { id: 'S20', kind: 'info', description: 'Search is disabled on some of your managed clusters.' },
+    ]
+    const { getByText } = render(
+        <MockedProvider mocks={[]}>
+            <HeaderWithNotification queryMessages={disableSearch} />
+        </MockedProvider>
+    )
+    expect(getByText).toMatchSnapshot()
```

Comment:
Update tests to validate the rendered element. Will need to update snapshot.
Let's talk an I can explain the reason.
```suggestion
    const { baseElement } = render(<HeaderWithNotification queryMessages={disableSearch} />)
    expect(baseElement).toMatchSnapshot()
```